contracts/Oracle/SovrynSwapPriceFeed.sol

```diff
@@ -1,12 +1,13 @@
 // SPDX-License-Identifier: MIT
 pragma solidity 0.8.13;
 
+import "@openzeppelin/contracts/token/ERC20/IERC20.sol";
```

Comment:
We are already using SovrynSwapPriceFeed today, how those changes are gonna affect it? will we upgrade it to this updated and upgraded contract?

Reply:
I don't think we need to upgrade it. To be honest, I could have kept it just using `adderss`, but it's more recommended to use `IERC20` (also Sovryn contracts uses that and not `address`). But it's the same in this manner so no need to upgrade.

Reply:
so what is the meaning of changing this code without deploying it?
it might be confusing for someone who reads this code and it's different from what is actually deployed also for us in the future we won't know this contract is not the one actually deployed.

Reply:
There will always be a gap between the time you develop something and the time you're deploying it. That's why we're managing https://github.com/BlindexDAO/BlinDEX/releases

With the approach, you're suggesting we'll always have to deploy right away after every small change we do, but there is no need for that, and it's even riskier that way.
We should manage releases of contracts and not deploy after every change